src/libstd/fs.rs

```diff
@@ -1299,6 +1299,20 @@ impl DirBuilder {
 
     /// Create the specified directory with the options configured in this
     /// builder.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// #![feature(dir_builder)]
+    /// use std::path::Path;
+    ///
+    /// let path = Path::new("/tmp/foo/bar/baz");
```

Comment:
This takes `P: AsRef<Path>` so idiomatically with a string literal you wouldn't also wrap it up in a `Path` but would instead just pass it through.